### PR TITLE
PE-581 Change the TF Unique ID of the SSO from email to SF Contact Id

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description 
 TF to start using Salesforce ContactID as the Unique Identifier

### CODE REVIEW

File: common/djangoapps/third_party_auth/management/commands/update_uid.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    example:
+        manage.py ... update_uid google-oauth2 salesforceid
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('provider')
+        parser.add_argument('field')
+
+    @transaction.atomic
+    def handle(self, provider, field, *args, **kwargs):
```
Comment (Squirrel18) [inline common/djangoapps/third_party_auth/management/commands/update_uid.py:32]: Unused argument args and kwargs. @andrey-canon

File: lms/djangoapps/instructor/views/tools.py [hunk 2]
```diff
@@ -71,7 +72,11 @@ def get_student_from_identifier(unique_student_identifier):
 
     DEPRECATED: use student.models.get_user_by_username_or_email instead.
```
Comment (Squirrel18) [inline lms/djangoapps/instructor/views/tools.py:73]: What about this message? Is it still valid? Just asking. @andrey-canon

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: common/djangoapps/third_party_auth/management/commands/update_uid.py (hunk 1 above)
Comment (andrey-canon) [reply common/djangoapps/third_party_auth/management/commands/update_uid.py:32]: the method  handle is called with more arguments, if I remove them, an exception is raised with the message `got an unexpected keyword argument ..........`

File: lms/djangoapps/instructor/views/tools.py (hunk 2 above)
Comment (andrey-canon) [reply lms/djangoapps/instructor/views/tools.py:73]: I don't know, this uses `student.models.get_user_by_username_or_email`

File: common/djangoapps/third_party_auth/management/commands/update_uid.py (hunk 1 above)
Comment (Squirrel18) [reply common/djangoapps/third_party_auth/management/commands/update_uid.py:32]: Got it.

**Comment by Squirrel18** [comment]
Squash and merge. @andrey-canon

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**